packages/patterns-react/src/patterns/PictogramArray/PictogramArrayItem.js

```diff
@@ -9,12 +9,27 @@ import { ArrowRight20 } from '@carbon/icons-react';
 import { LinkWithIcon } from '@carbon/ibmdotcom-react';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import {
+  settings as ddsSettings,
+  markdownToHtml,
+} from '@carbon/ibmdotcom-utilities';
 import { settings } from 'carbon-components';
 
 const { stablePrefix } = ddsSettings;
 const { prefix } = settings;
 
+/**
+ * Converts markdown to html and returns object to be used in `dangerouslySetInnerHTML`
+ *
+ * @param {string} string string to be converted to html
+ * @returns {object} object with a __html key containing converted string
+ */
+const convertMarkdown = string => ({
+  __html: markdownToHtml(string, {
+    italic: true,
+  }),
```

Comment:
@emerson-pereira as discussed, update the utility so that this would be `bold: false` instead.

Reply:
@jeffchew By default every style is false. The only way of limiting is opting-in. At least that's how the converter is designed right now.

Reply:
@emerson-pereira can the utility be updated to be opt-out instead?

Reply:
@jeffchew Sure, I'll work on it.

Reply:
@jeffchew Created PR #977

Reply:
@jeffchew updated to `bold: false`